localstack-core/localstack/services/apigateway/helpers.py

```diff
@@ -815,7 +820,7 @@ def create_method_resource(child, method, method_schema):
         kwargs = {}
 
         if authorizer := get_authorizer(method_schema) or default_authorizer:
-            method_authorizer = authorizer or default_authorizer
```

Comment:
@bentsku This seemed to be a redundant operation... Do you see any reason to keep this? Was this a issue with the minifier/obfuscation tool?

Reply:
Seems like this was just being too safe probably? Don't think it's because of the minifier, as this is community and this code isn't minified. Probably just a mistake not caught in a review 😄